aries_cloudagent/holder/routes.py

```diff
@@ -118,7 +118,7 @@ async def credentials_get(request: web.BaseRequest):
         The credential response
 
     """
-    context = request.app["request_context"]
+    context = request["context"]
```

Comment:
all requests now use context from the request object. This allows each admin request to have a different context